packages/jwt-verifier/test/integration-test/resources/testRunner.yml

```diff
@@ -17,21 +17,10 @@ scenarios:
   implicit-flow-local-validation:
     disabledTests:
       - wrongScopeAccessTokenTest
+      - scopeAccessTokenTest
```

Comment:
This test will be enabled after a new TCK version is released. It was failing due to an error in groovy test. Test updated but TCK jar needs to bumped up.